### PR TITLE
fix(explorer): sticky header needs more space

### PR DESCRIPTION
**Pull request by alber70g**

### CODE REVIEW

File: packages/apps/explorer/src/components/layout/styles.css.ts [hunk 1]
```diff
@@ -16,5 +16,6 @@ export const layoutWrapperClass = style({
 });
 
 export const contentClass = style({
+  height: '64px',
```
Comment (sstraatemans) [inline packages/apps/explorer/src/components/layout/styles.css.ts:19]: need to use a token for this

File: packages/apps/explorer/src/components/layout/layout.tsx [hunk 2]
```diff
@@ -14,7 +14,6 @@ export const Layout: FC<IProps> = ({ children }: IProps) => {
   return (
     <div className={documentStyle}>
       <Header />
-      <Stack marginBlock="xl" />
       <Stack
```
Comment (sstraatemans) [inline packages/apps/explorer/src/components/layout/layout.tsx:17]: you can add the marginblockstart to the below stack

**Review by sstraatemans — APPROVED** [review]

File: packages/apps/explorer/src/components/layout/layout.tsx (hunk 2 above)
Comment (alber70g) [reply packages/apps/explorer/src/components/layout/layout.tsx:17]: Isa mentioned that the header height isn't tokenized...

File: packages/apps/explorer/src/components/layout/styles.css.ts (hunk 1 above)
Comment (alber70g) [reply packages/apps/explorer/src/components/layout/styles.css.ts:19]: Isa mentioned that the header height isn't tokenized...

**Pull request merged**